Labs/Alyssa_Giedd_Lab2.ipynb

```diff
@@ -0,0 +1,477 @@
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x22710e71b38>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "#B) If you have to look for a signal in 10k pixels, what is the probability distribution of your background? \n",
+    "#(Clearly state the statistical question, then turn that into math using your background distribution from part 1 of this problem.)\n",
+    "print(\"What is the likelihood that the background gave me a signal as signal-like or more than the observed value?\")\n",
+    "plt.plot(x, bg*10000)\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 83,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Sigma value for 10k pixels at 18 = 4.267904750513104\n"
+     ]
+    }
+   ],
+   "source": [
+    "#C) Taking your brightest candidate signal from the region (assume it has the same signal as in part 1), \n",
+    "#calculate the significance of your detection.\n",
+    "probability = (1 - stats.norm.cdf(18, loc = 0, scale = 3))*10000\n",
+    "sigma = stats.norm.ppf(1 - probability)\n",
+    "print(\"Sigma value for 10k pixels at 18 =\", sigma)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Problem 4"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 91,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Version 1's 5-sigma detection would require a signal of 14.999999999910528\n",
+      "Version 2's 5-sigma detection would require a signal of 19.651612460674965\n"
+     ]
+    }
+   ],
+   "source": [
+    "#A) Calculate the signal required for a 5-sigma detection in Version 1\n",
+    "pixelct1 = stats.norm.cdf(5)\n",
+    "signal = stats.norm.ppf(pixelct1, scale=3)\n",
+    "\n",
+    "print(\"Version 1\\'s 5-sigma detection would require a signal of\", signal)\n",
+    "\n",
+    "#B) Calculate the signal required for a 5-sigma detection in Version 2\n",
+    "pixelct10k = 1 - stats.norm.cdf(5)\n",
+    "signal_10k = stats.norm.ppf(1 - pixelct10k/10000, scale = 3)\n",
+    "\n",
+    "print(\"Version 2\\'s 5-sigma detection would require a signal of\", signal_10k)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 99,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "1.5617614993929534\n",
+      "An increase in signal would need to be approximately 30% brighter to meet the required threshold. \n",
```

Comment:
your ratio was 1.56, i think that would correspond to 56% brighter instead of the 30% that you print in the next line